src/server/data-access/session.ts

```diff
@@ -3,27 +3,24 @@ import "server-only";
 import { routes } from "@/common/routes";
 import { HOST_URL } from "@/server/config/server.config";
 import {
-  createClient,
-  getOAuthProvider,
-  createAdminClient,
-  createSessionClient,
   getUniqueID,
   oAuthProvidersType,
 } from "@/server/services/appwrite";
+import { serviceClient } from "../services";
 
 export async function getSession() {
-  const { account } = await createSessionClient();
+  const account = (await serviceClient.createSession()).account;
```

Comment:
`const { account } = await serviceClient.createSession();`

is this not supported